root/02-services.html

```diff
@@ -62,8 +62,17 @@
 
      <h1>MEET OUR SERVICES</h1>
 
+      <p>Software development</p>
+      <img src="E:\xavi downloads\soft ware 1.avif" alt="software image">
```

Comment:
We may use single names in the names, e.g. for the alt attribute as `software-image` or `software-img`